.github/ISSUE_TEMPLATE/documentation-needed.md

```diff
@@ -1,17 +1,30 @@
 ---
 name: Documentation Needed
-about: Describe this issue template's purpose here.
-title: "[Documentation]"
+about: Proposal for additional documentation or examples of code usage
+title: "[Doc]"
 labels: documentation
 assignees: ''
 
 ---
 
-## Description 
-A clear and concise description of what is missing in documentation
+# Description 
 
-## Impact 
-Description of the current or possible impact of this documentation
+A clear and concise description of what is missing in documentation, or how documentation may be improved. 
+What failings have occurred due to this lack of information?
 
-## Proposed solutions
-_What would you recommend to fix it?_
+# Impact 
+
+Description of the current or possible impact of this addition
+
+# Proposed solutions
+
+Do you have a solution, what would you recommend to fix it?
```

Comment:
This is awkwardly worded / punctuated. For whatever reason an alternative is not immediately springing to mind, but lets do better.

Reply:
how about 
> Do you have a recommend solution that may fix this issue?